hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3275,6 +3275,24 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.client.ec.enable.retries</name>
+    <value>true</value>
+    <tag>CLIENT</tag>
+    <description>
+      To enable Grpc client retries for EC.
+    </description>
+  </property>
+
+  <property>
+    <name>ozone.client.ec.grpc.max.retries</name>
```

Comment:
I'd prefer

 * `ozone.client.ec.grpc.retries.enabled`
 * `ozone.client.ec.grpc.retries.max`

to keep these more closely related.  (Please feel free to drop `grpc` from both if you wish.)

Reply:
Thanks for the suggestion. Will update the PR. Adding `grpc` makes sense as this is specific for EC grpc.